roles/openshift_logging/defaults/kibana.yml

```diff
@@ -0,0 +1,3 @@
+---
+kibana_hostname: kibana.example.com
+kibana_ops_hostname: kibana-ops.example.com
```

Comment:
We have a variable for setting the master config (openshift_master_logging_public_url), I don't necessarily want to overload the use of the openshift_master prefix here, but what if we used `openshift_logging_public_url` and then handle defaulting those in the playbooks properly? Then these could become:
`"kibana.{{ openshift_logging_public_url }}"` and `"kibana-ops.{{ openshift_logging_public_url }}"`